NitroxClient/MonoBehaviours/AnimationController.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using UnityEngine;
+
+namespace NitroxClient.MonoBehaviours
+{
+    public class AnimationController : MonoBehaviour
+    {
+        public static readonly float SMOOTHING_SPEED = 0.95f;
```

Comment:
Will this constant be the same for all animations?  For the future(not this commit) may need to adjust this based on latency.